db/db_impl/db_impl.cc

```diff
@@ -2084,9 +2085,29 @@ Status DBImpl::GetEntity(const ReadOptions& _read_options, const Slice& key,
   std::vector<Slice> keys;
   std::vector<ColumnFamilyHandle*> column_families;
   for (size_t i = 0; i < num_column_families; ++i) {
-    // Adding the same key slice for different CFs
-    keys.emplace_back(key);
-    column_families.emplace_back((*result)[i].column_family());
+    if (!(*result)[i].column_family()) {
+      if (s.ok()) {
+        s = Status::InvalidArgument(
+            "DB failed to query because one or more group(s) have null column "
+            "family handle");
+      }
+      (*result)[i].SetStatus(
+          Status::InvalidArgument("Column family handle cannot be null"));
```

Comment:
If there are more than one nullptr CFHs, we will only detect the first and the rest will have `Status::Incomplete()`. I think that should be fine since we are failing the query entirely. Let me make the change.